### PR TITLE
[chore] Some logging updates for feeds aggregator

### PR DESCRIPTION
**Pull request by cedric-cordenier**

### CODE REVIEW

File: pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go [hunk 1]
```diff
@@ -105,28 +104,38 @@ func (a *dataFeedsAggregator) Aggregate(previousOutcome *types.AggregationOutcom
...
+	lggr.Debugw("determined feeds to check", "nFeedIds", len(allIDs))
 	// ensure deterministic order of reportsNeedingUpdate
 	sort.Slice(allIDs, func(i, j int) bool { return allIDs[i] < allIDs[j] })
 	candidateIDs := []string{}
 	for _, feedIDStr := range allIDs {
 		previousReportInfo := currentState.FeedInfo[feedIDStr]
 		feedID, err2 := datastreams.NewFeedID(feedIDStr)
 		if err2 != nil {
-			a.lggr.Errorf("could not convert %s to feedID", feedID)
+			lggr.Errorf("could not convert %s to feedID", feedID)
```
Comment (ettec) [inline pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go:115]: would it be beter to be consistenet with feed id here and use Errorw?

**Review by ettec — APPROVED** [review]

**Review by agparadiso — APPROVED** [review]

File: pkg/capabilities/consensus/ocr3/aggregators/identical.go [hunk 2]
```diff
@@ -36,18 +36,18 @@ type counter struct {
 
 var _ types.Aggregator = (*identicalAggregator)(nil)
 
-func (a *identicalAggregator) Aggregate(_ *types.AggregationOutcome, observations map[ocrcommon.OracleID][]values.Value, f int) (*types.AggregationOutcome, error) {
+func (a *identicalAggregator) Aggregate(lggr logger.Logger, _ *types.AggregationOutcome, observations map[ocrcommon.OracleID][]values.Value, f int) (*types.AggregationOutcome, error) {
```
Comment (jmank88) [inline pkg/capabilities/consensus/ocr3/aggregators/identical.go:39]: Passing a logger as method arguments is generally a code smell. Why is this necessary, when `identicalAggregator` already has an internal logger?

Comment (jmank88) [reply pkg/capabilities/consensus/ocr3/aggregators/identical.go:39]: @cedric-cordenier What is going on here? The linter is flagging the unused field now in @chudilka1's PR
